Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "soda-test-service"
-version = "0.3.0"
+version = "0.3.3"

```

```suggestion
version = "0.3.1"
```

Not sure why you are patching 3 versions in one... If you already pushed a private version of the test-service with 2 other patches you will need to yank them and publish the `0.3.1` patch again.